index.html

```diff
@@ -714,8 +724,14 @@ <h3>
             </ol>
           </li>
           <li>
-            <a>Queue a task</a> to <a>fire an event</a> named "`release`" at
-            |lock|.
+            <a>Queue a task</a> to run the following steps:
+            <ol>
+              <li>Set |lock|.{{WakeLockSentinel/released}} to `true`.
```

Comment:
Depends how 'correctly' you want to spec this, but here you're setting a readonly value. To avoid this ambiguity, I would define `.released` as a getter, which returns some internal value. Then you can set the internal value, which makes it clear that it doesn't go via JS setters etc. Eg https://wicg.github.io/background-fetch/#background-fetch-record-interface

Reply:
Thanks for the suggestions! I've tried to incorporate them in #280, please have a look.